src/driver-hidpp20.c

```diff
@@ -1305,6 +1306,9 @@ hidpp20drv_init_feature(struct ratbag_device *device, uint16_t feature)
 		break;
 	}
 	case HIDPP_PAGE_COLOR_LED_EFFECTS: {
+		if (ratbag_device_data_hidpp20_get_quirk(device->data) == HIDPP20_QUIRK_G602)
```

Comment:
needs a short comment explaining why this needs to be done, if for no other reason to make it obvious whether the code does what we intend do do.